test/compare-acorn-es6/class-declaration.expected.js

```diff
@@ -0,0 +1,12 @@
+class ComputedKey {
+    [n1 + n2]() {
+    }
+}
+class ComputedKeyWithParenthesis {
+    [(n1 + n2) * n3]() {
```

Comment:
A more helpful test would use an assignment (testing the other boundary condition) or a `yield`.

Reply:
Not sure what did you meant.

Reply:
Did you meant this?
189e8c5